script/type_annotations.lua

```diff
@@ -73,15 +73,19 @@
 ---@class Job
 ---@field active boolean?
 ---@field action JobAction
----@field action_args table<any>?
+---@field action_args table, <any>?
 ---@field leave_condition JobLeaveCondition
----@field leave_args table<any>?
----@field constructrons LuaEntity[]
+---@field leave_args table, <any>?
```

Comment:
I believe the `,` here is wrong but I'll have a look at that in the future when I get to improving the annotations a bit more